apps/prairielearn/src/pages/partials/jobSequenceResults.ejs

```diff
@@ -68,12 +68,14 @@
     <% job_sequence.jobs.forEach(function(job) { %>
     <div class="list-group">
         <li class="list-group-item">
-            <h4 class="list-group-item-heading">Stage <%= job.number_in_sequence %>: <%= job.description %></h4>
-            <% if (job.command) { %>
-            <p class="mb-1">
-                Command <code><%= job.command %><% job.arguments.forEach(function(arg) { %> <%= arg %><% }); %></code>
-                <% if (job.working_directory) { %> run in <code><%= job.working_directory %></code><% } %>
-            </p>
+            <% if (job_sequence.legacy) { %>
+                <h4 class="list-group-item-heading">Stage <%= job.number_in_sequence %>: <%= job.description %></h4>
+                <% if (job.command) { %>
+                <p class="mb-1">
+                    Command <code><%= job.command %><% job.arguments.forEach(function(arg) { %> <%= arg %><% }); %></code>
+                    <% if (job.working_directory) { %> run in <code><%= job.working_directory %></code><% } %>
+                </p>
+                <% } %>
```

Comment:
I turned off printing of the command, working directory, status, and signal since those should never be set on the `job` itself for non-legacy sequences.